changes/ce/fix-14023.en.md

```diff
@@ -0,0 +1 @@
+Fixed an issue with the `GET /monitor` HTTP API where returned data could return values greater than reality depending on the requested time window.
```

Comment:
it only happens to data points older than 1h ?
```suggestion
Fixed an issue with the `GET /monitor` HTTP API where returned data could return values greater than reality for sampled metrics older than 1 hour.
```

Reply:
No, it happens to fresh points as well, as shown in the video.

Reply:
maybe we should make it clear in change log:
the data points older than 1h is permanently altered, the data points within 1h is only displayed wrong.

Reply:
:checkered_flag: